bin/runtime/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-runtime"
-version = "0.8.0+mainnet"
+version = "0.8.1"
```

Comment:
Is that  truly correct? This shouldn't be changing the runtime afaik, so this seems suspicious to me.

Reply:
You're right, reverted